hibernate-core/src/main/java/org/hibernate/query/sqm/sql/BaseSqmToSqlAstConverter.java

```diff
@@ -785,7 +785,7 @@ public Expression resolveSqlExpression(
 				if ( sqmAssignment.getValue() instanceof SqmParameter ) {
 					final SqmParameter<?> sqmParameter = (SqmParameter<?>) sqmAssignment.getValue();
 
-					consumeSqmParameter(
+					consumeSingleSqmParameter(
```

Comment:
Was this change on purpose?

Reply:
we renamed the method

Reply:
I know, but it seems like we might have forgotten to replace this one with `consumeSqmParameter` again. Nevermind though, it seems odd to have a multi-valued parameter in the SET clause.